src/backend/utils/storage.js

```diff
@@ -69,4 +70,39 @@ module.exports = {
   getPostsCount: () => redis.zcard(POSTS),
 
   getPost: async guid => redis.hgetall(guid),
+
+  /**
+   * Moves a feed from active(FEEDS) to inactive(INACTIVE_FEEDS), will check if the set exists first before moving
+   * @param feedId key of set being moved
+   * @return feedId of key being moved
+   */
+  addInactiveFeed: async feedId => {
+    if ((await redis.exists(INACTIVE_FEEDS)) === 1) {
+      await redis.smove(FEEDS, INACTIVE_FEEDS, feedId);
+    } else {
+      const name = await redis.hget(feedId, 'name');
+      const url = await redis.hget(feedId, 'url');
+      await redis
+        .multi()
+        // Using hmset() until hset() fully supports multiple fields:
+        // https://github.com/stipsan/ioredis-mock/issues/345
+        // https://github.com/luin/ioredis/issues/551
+        .hmset(feedId, 'name', name, 'url', url)
+        .sadd(INACTIVE_FEEDS, feedId)
+        .exec();
+    }
+    return feedId;
+  },
+
+  /**
+   * Gets an array of guids from redis
+   * @param from lower index
+   * @param to higher index, it needs -1 because redis includes the element at this index in the returned array
+   * @return Array of guids
+   */
```

Comment:
Do you need this comment here? You're not passing anything to 'getInactiveFeeds` so I don't know what that comment is for.

Reply:
Hahaha, you're right. I had fixed this in a new commit but didn't push it to origin. I just wanted some thought on it.